product_docs/docs/pgd/5.6/reference/commit-scopes.mdx

```diff
@@ -107,22 +123,42 @@ This is the default visibility. A transaction is confirmed after all of its chan
 
 More details of the commit scope kinds and details of their parameters:
 
+-   [Synchronous Commit](#synchronous_commit)
```

Comment:
Why the reordering? And why not GROUP COMMIT as lead?

Reply:
I was thinking SYNCHRONOUS COMMIT as lead because it is simpler and seems to be preferred for use (it is the default commit scope for Big Animal for instance).